### PR TITLE
add comment in changelog

### PR DESCRIPTION
**Pull request by Jessica-Koch**
correct a changelog entry

### CODE REVIEW

File: change/@microsoft-teams-js-2519cd0a-0e06-49b5-b79d-338d4042a1a3.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "type": "patch",
   "packageName": "@microsoft/teams-js",
+  "comment": "update comment for initializePrivateApis explaining that this function needs to stay for backwards compatibility purposes",
```
Comment (erinha) [inline change/@microsoft-teams-js-2519cd0a-0e06-49b5-b79d-338d4042a1a3.json:4]: Can you please capitalize the first word "Update"?

Comment (erinha) [reply change/@microsoft-teams-js-2519cd0a-0e06-49b5-b79d-338d4042a1a3.json:4]: Is "initializePrivateApis" a function?  If so, can you update and format it like this: "`initializePrivateApis()` function"

**Review by erinha — CHANGES_REQUESTED** [review]
Thanks for fixing this! Small updates requested please.

File: change/@microsoft-teams-js-2519cd0a-0e06-49b5-b79d-338d4042a1a3.json (hunk 1 above)
Comment (Jessica-Koch) [reply change/@microsoft-teams-js-2519cd0a-0e06-49b5-b79d-338d4042a1a3.json:4]: sure

**Review by erinha — APPROVED** [review]
Beautiful, thanks!

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**